aws-codeguruprofiler-profilinggroup/aws-codeguruprofiler-profilinggroup.json

```diff
@@ -45,6 +45,16 @@
         }
       }
     },
+    "ChannelUris": {
+      "description": "The Amazon Resource Name (ARN) of the specified SNS arn",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/Arn",
+        "examples": [
+          "arn:aws:codeguru-profiler:us-east-1:000000000000:profilingGroup/My-sns-arn"
```

Comment:
This should the SNS Topic ARN and not a profiling group ARN, e.g. `arn:aws:sns:us-east-2:123456789012:MyTopic`